sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -938,6 +964,139 @@ The output of the `sem delete notification documents` command is as follows:
     $ sem delete notification documents
     Notification 'documents' deleted.
 
+## Working with pipelines
+
+### Listing pipelines
+
+The `sem get pipelines` command returns the list of pipeline for a Semaphore
+2.0 project. If you are inside the directory of a Semaphore project, then you
+will not need to provide `sem` the name of the project as this will be
+discovered automatically. However, if there is a problem with the format of the
+GitHub URL of your repository or if you are outside the directory of a GitHub
+repository you can use the `-p` flag to declare the Semaphore project that
+interests you.
```

Comment:
Maybe we should provide more positive use-case.
I would replace:
"if there is a problem with the format of the
 GitHub URL of your repository or if you are outside the directory of a GitHub
 repository"
with
"if you want list of pipelines for some project other than the one you are currently in"

Reply:
Done.